docs/guide/migrating-to-version-9.md

```diff
@@ -414,6 +414,31 @@ This component has been removed. Please refer to KBadge component which has been
 
 KPagination now uses [KDropdown](/components/dropdown) instead of [KSelect](/components/select) as the underlying component for page size selection.
 
+#### Props
+
+* `testMode` prop has been removed
+* `paginationType` prop has been removed. You can use new `offset` prop instead
+* `offsetPrevButtonDisabled` prop has been renamed to `offsetPreviousButtonDisabled`
+
+#### Events
+
+* `pageChanged` event has been renamed to `pageChange`
+* `pageSizeChanged` event has been renamed to `pageSizeChange`
+* `getPrevOffset` event has been renamed to `getPreviousOffset`
+
+#### Structure
+
+* `card-pagination-bar` has been removed and replaced with `k-pagination`
+* `k-pagination-container` has been replaced with `k-pagination`
```

Comment:
So both of these were the same element?

Reply:
Not quite, `card-pagination-bar` was a wrapper div inside `k-pagination-container` but I found it to be unnecessary